clouds/clouds-aws/src/test/java/aws/cloudformation/di/suitetests/AwsEapHostnameTests.java

```diff
@@ -0,0 +1,51 @@
+package aws.cloudformation.di.suitetests;
+
+
+import aws.core.identification.AwsEc2Instance;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import sunstone.api.Parameter;
+import sunstone.api.WithAwsCfTemplate;
+import sunstone.api.inject.Hostname;
+
+import static aws.cloudformation.AwsTestConstants.instanceName;
+import static aws.cloudformation.AwsTestConstants.region;
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+/**
+ * Shared resources among other DI tests.
```

Comment:
Is this comment still valid. I am confused.

Reply:
Oh, you mean that it uses `perSuite = true`

Reply:
Yeah.